### PR TITLE
feat: new ConditionsExpression Enum move dynamo functions into dynamo folder

### PR DESCRIPTION
**Pull request by gabrielforster**
### Description
In this pr I create a new ConditionsExpression Enum. It is a enum with all valid conditions for dynamo queries.
I also move dynamo structure into a dynamo folder.

### CODE REVIEW

**Review by CauaKath — APPROVED** [review]
LGTM

File: src/dao/dynamo/dao.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { isObject } from '../object'
+import { isObject } from '../../object'
 import dbClient from 'aws-dynamodb-factory-js'
-import { error } from '../error'
-import type { DynamodbParams, DynamodbResponseBatch } from './interfaces'
+import { error } from '../../error'
+import type { DynamodbParams, DynamodbResponseBatch } from '../interfaces'
```
Comment (caroltrindade) [inline src/dao/dynamo/dao.ts:4]: I think you can move dynamo interfaces to dynamo/types as well

**Review by caroltrindade — APPROVED** [review]

File: src/dao/dynamo/dao.ts (hunk 1 above)
Comment (caroltrindade) [inline src/dao/dynamo/dao.ts:4]: I think you can move dynamo interfaces to dynamo/types as well

**Review by caroltrindade — CHANGES_REQUESTED** [review]

File: src/dao/dynamo/dao.ts (hunk 1 above)
Comment (gabrielforster) [reply src/dao/dynamo/dao.ts:4]: Sure. I'll do it!

**Review by caroltrindade — APPROVED** [review]
LGTM

**Pull request merged**